lib/simplecov/source_file.rb

```diff
@@ -82,8 +82,7 @@ def initialize(filename, coverage)
 
     # The path to this source file relative to the projects directory
     def project_filename
-      project_dir = File.dirname(File.expand_path(File.basename(__FILE__)))
-      @filename.sub(/^#{project_dir}/, "")
+      @filename.sub(/^#{SimpleCov.root}/, "")
```

Comment:
hm not too sure but I think it looks good :+1: 

In the future though, would be preferred to have unrelated changes for other issues etc. in their own PR so they don't get too big and stay more focussed.